app/js/actions/artifacts.js

```diff
@@ -119,8 +119,7 @@ export const refreshArtifacts = () => {
         .then((json) => {
             json.artifacts.forEach(artifact => dispatch(newArtifact(artifact)));
         })
-        .then(() => dispatch(actions.checkTypes()))
-        .then(() => dispatch(actions.checkImportableTypes()));
```

Comment:
This was a mistake from my previous PR, we don't need this function call as it already happens in `establishConnection` per @ebolyen 's idea in his code review of that PR.